src/vsr/journal.zig

```diff
@@ -1303,6 +1279,33 @@ pub fn JournalType(comptime Replica: type, comptime Storage: type) type {
                 });
             }
 
+            // Discard headers which we are certain do not belong in the current log_view.
```

Comment:
Is there some comment somewhere which explains how prepares with a wrong view can arise in the first place?

Reply:
I'll add this explanation:
```
            // (These headers can originate if we join a view, write some prepares from the new
            // view, and then crash before the view_durable_update() finished.)
```